impl/sql/mutationstorage/queue.go

```diff
@@ -51,6 +51,7 @@ func (m *Mutations) NewReciever(ctx context.Context, last time.Time, mapID, star
 		ticker:      time.NewTicker(ropts.Period),
 		maxTicker:   time.NewTicker(ropts.MaxPeriod),
 		done:        make(chan interface{}),
+		more:        make(chan bool, 1),
```

Comment:
it does seem a bit inconsistent for these two signalling channels `done` and `more` to be of differing types.  They are fairly equivalent aren't they?

Reply:
I never try to put anything in `done`. 
`more` is actually more equivalent to a ticker which is `chan time.Time`, so I've made it a channel of `time.Time` for consistency.